app/models/user.rb

```diff
@@ -3,4 +3,7 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
+
+  has_many :foods, dependent: :destroy
+  has_many :recipes, dependent: :destroy
```

Comment:
These lines create a conflict with the dev branch.

Reply:
It has been resolved